Makefile

```diff
@@ -22,126 +12,170 @@ DRIVER_USER=sqlboiler_driver_user
 DRIVER_DB=sqlboiler_driver_test
 
 # Builds all software and runs model tests
+.PHONY: tests
 tests: test-psql test-mysql test-mssql
+
 # Creates super users
+.PHONY: test-users
 test-users: test-user-psql test-user-mysql test-user-mssql
+
 # Creates databases
+.PHONY: test-dbs
 test-dbs: test-db-psql test-db-mysql test-db-mssql
 
 # Runs tests on the drivers
+.PHONY: driver-tests
 driver-tests: driver-test-psql driver-test-mysql driver-test-mssql
+
 # Creates regular databases
+.PHONY: driver-dbs
 driver-dbs: driver-db-psql driver-db-mysql driver-db-mssql
+
 # Creates regular users with access to only one DB, may modify the DB
 # to ensure write access to the user
+.PHONY: driver-users
 driver-users: driver-user-psql driver-user-mysql driver-user-mssql
 
-run-mssql:
-	docker run --detach --rm --env 'ACCEPT_EULA=Y' --env 'SA_PASSWORD=Sqlboiler@1234' --publish 1433:1433 --name mssql microsoft/mssql-server-linux:2017-latest
-kill-mssql:
-	docker rm --force mssql
-
 # ====================================
 # Building operations
 # ====================================
 
+.PHONY: build
 build:
 	go build github.com/volatiletech/sqlboiler
 
+.PHONY: build-psql
 build-psql:
 	go build github.com/volatiletech/sqlboiler/drivers/sqlboiler-psql
 
+.PHONY: build-mysql
 build-mysql:
 	go build github.com/volatiletech/sqlboiler/drivers/sqlboiler-mysql
 
+.PHONY: build-mssql
 build-mssql:
 	go build github.com/volatiletech/sqlboiler/drivers/sqlboiler-mssql
 
 # ====================================
 # Testing operations
 # ====================================
 
-test-psql: build build-psql
-	./sqlboiler --wipe psql
-	go test ./models
-
-test-mysql: build build-mysql
-	./sqlboiler --wipe mysql
-	go test ./models
-
-test-mssql: build build-mssql
-	./sqlboiler --wipe mssql
-	go test ./models
+.PHONY: test
+test:
+	go test -v -race $(go list ./... | grep -v /drivers/ | grep -v /vendor/)
 
+.PHONY: test-user-psql
 test-user-psql:
 	# Can't use createuser because it interactively promtps for a password
-	psql --command "create user $(USER) with superuser password '$(PASS)';" postgres
+	psql --host localhost --username postgres -c "create user $(USER) with superuser password '$(PASS)';"
```

Comment:
The expansion of `--command` here was intentional. In all scripts it's generally better to use the expanded form of an argument. The `-c` is to save you typing once you know it when executing via the command line, but when we only have to read it because it's in a script, it becomes better to have the full argument visible to the reader since quite often you won't require a trip to the manpage if you see the full name. (And yea, I'm aware that `-c` is a common "run command" argument in shells etc, I don't think that makes the argument weaker).

Reply:
Ok.